build-system/pr-check/local-tests.js

```diff
@@ -79,18 +79,17 @@ function main() {
     ) {
       timedExecOrDie(
         'gulp integration --nobuild --headless --coverage --report'
-      ); // DO NOT SUBMIT
+      );
     }
 
     if (buildTargets.has('RUNTIME') || buildTargets.has('UNIT_TEST')) {
-      timedExecOrDie('gulp unit --nobuild --headless --coverage --report'); // DO NOT SUBMIT
+      timedExecOrDie('gulp unit --nobuild --headless --coverage --report');
```

Comment:
Is this still the right flag?

Reply:
Yup! I can rename it to `--test-report`, but then I'd have to do `argv['test-report']` instead of `argv.report`, and I think it's clear(ish) from the context. I see how this could collide if we wanted to add other kinds of reports down the line, but that's down the line -- what do you think?

Reply:
Just double-checking cause I remember something being renamed in another PR, but I think it was the name of the gulp task itself